queue/abstract/driver/fifo.lua

```diff
@@ -66,9 +66,36 @@ function method.normalize_task(self, task)
     return task
 end
 
+-- check if mvcc is enabled
+local function check_mvcc_state()
+    local cfg = box.cfg
+    if cfg['memtx_use_mvcc_engine'] == nil  or cfg['memtx_use_mvcc_engine'] == false then
+        return false
+    end
+
+    return true
+end
```

Comment:
Just `if box.cfg.memtx_use_mvcc_engine then <...> end` is equivalent and simpler.

Reply:
changed function